.github/workflows/tag.yml

```diff
@@ -16,7 +24,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.BROADBOT_TOKEN }}
           HOTFIX_BRANCHES: hotfix.*
-          DEFAULT_BUMP: minor
+          DEFAULT_BUMP: patch
```

Comment:
I'm not clear how this works-- can you explain a bit more? What will the automatic tag numbers be? If we need to change it to be something/specific (major release for instance), how do we do that?

Reply:
Would one manually create a tag and then manually run a build? I know how to create a tag in github, but I'm not clear how to trigger the build step.

Reply:
You can manually trigger any GitHub workflow that has a workflow event dispatch trigger, using the Run workflow dropdown on the workflow page: https://github.com/DataBiosphere/terra-billing-profile-manager/actions/workflows/tag.yml

In the case of this workflow (tag), you typically wouldn't run it manually. It's run a from the `build and test` workflow if that workflow was run on the `main` branch, which normally happens after a PR is merged.

Regarding `minor` vs `patch`; we started out using `patch` but then changed it as we reasoned that `minor` would be used for regular (non-major) releases, and `patch` would be used for patches only, where the only change was to fix an issue in a previous version.

To control the version number used for the release, you can use a commit message to override the `DEFAULT_BUMP` behavior. See https://github.com/DataBiosphere/github-actions/blob/master/actions/bumper/README.md#bumping

Reply:
Thanks @pshapiro4broad . @marctalbott do you think we should stick with `minor` here?

Reply:
`patch` felt more natural to me as it's what we use in WSM and workbench-libs, so that's why I changed it. I don't feel particularly strongly about it though, so we can revert to `minor` if that's preferred

Reply:
(Regarding `minor` vs `patch`, both were discussed on our team so I wasn't trying to say what you have is incorrect, I wanted to let you know the thinking behind what we did.)

Reply:
I don't really know which one we want, but it's easy enough to change later if we wish.